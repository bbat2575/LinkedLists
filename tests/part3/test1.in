NEW 0
NEW 1
1
NEW 2
1
2.2
NEW 3
@
one
two
NEW 3
{0}
1
2
VIEW 4
TYPE 4
NEW 3
{0}
1
{2}
VIEW 5
TYPE 5
NEW 3
{0}
{1}
{2}
VIEW 6
TYPE 6
NEW 6
{0}
2e6
{1}
hmm
{2}
#
VIEW 7
TYPE 7
