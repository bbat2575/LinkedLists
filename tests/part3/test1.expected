
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 

> List 1: 1

> List 2: 1 -> 2.20

> List 3: @ -> one -> two

> Nested 4: {List 0} -> 1 -> 2

> {List 0} -> 1 -> 2

> reference -> int -> int

> Nested 5: {List 0} -> 1 -> {List 2}

> {List 0} -> 1 -> {List 2}

> reference -> int -> reference

> Nested 6: {List 0} -> {List 1} -> {List 2}

> {List 0} -> {List 1} -> {List 2}

> reference -> reference -> reference

> Nested 7: {List 0} -> 2000000.00 -> {List 1} -> hmm -> {List 2} -> #

> {List 0} -> 2000000.00 -> {List 1} -> hmm -> {List 2} -> #

> reference -> float -> reference -> string -> reference -> char

> 
